src/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>To-do-List</title>
+</head>
+
+<body>
+    <div class="to-do">
+        <div class="today-todo">
+            <p>Today's to do</p>
+            <i class="fas fa-sync"></i>
+        </div>
+        <div class="text">
+            <input type="text" class="inputTask" placeholder="Add to your list...">
+            <button class="submit" type="submit">↵</button>
+        </div>
+        <ul class="listItems">
+        </ul>
+        <button class="clear" type="button">Clear all completed</button>
+    </div>
```

Comment:
I highly recommend you to use semantic tags like main, section and form instead of all divs.

Reply:
Done :+1: